Makefile

```diff
@@ -481,6 +481,16 @@ themispp_uninstall:
 	@echo -n "themispp uninstall "
 	@$(BUILD_CMD_)
 
+rustthemis_examples_install: CMD = cargo install --examples --path .
```

Comment:
Maybe put these into `test.mk`? 

I'm a bit concerned that main makefile has example installation for Rust

Reply:
Yeah, that makes sense. Furthermore, we do not need _all_ examples here, just the ones which are needed for integration testing.

I'll move the Makefile target to `test.mk`.

Reply:
great!